### PR TITLE
Add more conditions for the lint ci and code coverage

### PR DESCRIPTION
**Pull request by Aisuko**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Aisuko** [comment]
Hi, guys. Thanks for your support. It looks like all the CI in this repo is working well.

File: .github/workflows/approve-to-run-ci.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        # golandci-lint need to be run before codecov and succeed
+        needs: golangci-lint
+        name: Code Coverage
+        if: github.repository == 'meshery/meshery-operator'
+        runs-on: ubuntu-22.04
+        steps:
+          - name: Checkout code
+            uses: actions/checkout@master
+          - name: Set up Go
+            uses: actions/setup-go@v2
```
Comment (acald-creator) [inline .github/workflows/approve-to-run-ci.yml:37]: I believe `actions/setup-go` is on `v3` going on to `v4`

File: .github/workflows/approve-to-run-ci.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    # if review comment includes approved or LGTM, run the tests
+    build:
+        name: Build
+        if: github.event_name == 'pull_request' && (contains(github.event.pull_request.body, 'approved') || contains(github.event.pull_request.body, 'LGTM'))
+        runs-on: ubuntu-22.04
+        steps:
+          - name: Checkout code
+            uses: actions/checkout@master
+          - name: Set up Go
+            uses: actions/setup-go@v2
```
Comment (acald-creator) [inline .github/workflows/approve-to-run-ci.yml:59]: same here

File: .github/workflows/approve-to-run-ci.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+jobs:
+    golangci-lint:
+        strategy:
+          matrix:
+            platform: [ubuntu-22.04]
+            go-version: [1.19.x]
+        runs-on: ${{ matrix.platform }}
+        steps:
+          - uses: actions/setup-go@master
```
Comment (acald-creator) [inline .github/workflows/approve-to-run-ci.yml:17]: also this one is using master, i think it would be best to keep it consistent

**Review by acald-creator — CHANGES_REQUESTED** [review]
see comments

File: .github/workflows/approve-to-run-ci.yml (hunk 1 above)
Comment (Aisuko) [reply .github/workflows/approve-to-run-ci.yml:37]: Nice catch, I will replace it to `v4`

**Comment by Aisuko** [comment]
Sorry, not sure why it includes so many files and was auto-closed. And I cannot reopen it. I will open a new one. Thanks.

**Comment by Aisuko** [comment]
@acald-creator All changes were done at PR#471, please take a look.

**Pull request closed without merging**